xsgen/openmc_origen.py

```diff
@@ -564,8 +598,9 @@ def _generate_xs(self, e_g, phi_g):
         rc = self.rc
         verbose = rc.verbose
         xscache = self.xscache
-        xscache['E_g'] = e_g
-        xscache['phi_g'] = phi_g
+        xscache.clear()
+        #xscache['E_g'] = e_g
+        #xscache['phi_g'] = phi_g
```

Comment:
Yeah, those should not be removed. I commented them out because they were failing. PyNE does not like those calls. I've made some changes to PyNE so that they work.